demo/angular-editable-text.js

```diff
@@ -10,31 +10,32 @@
 
 })();
 
-/**
- * Created by Gabriel Grinberg on 6/13/14.
- */
+/*
+   Created by Gabriel Grinberg on 6/13/14.
+   */
 
-(function () {
+(function() {
   'use strict';
-  angular.module('gg.editableText')
-    .directive('editableText', ['EditableTextHelper', function (EditableTextHelper) {
+  angular.module('gg.editableText', ['puElasticInput'])
+    .directive('editableText', ['$timeout', 'EditableTextHelper', function($timeout, EditableTextHelper) {
       return {
         scope: {
           editableText: '=',
           editMode: '=',
           placeholder: '@',
-          onChange: '&'
+          onChange: '&',
+          onReject: '&',
```

Comment:
I agree with the comment, but it might make a PR back to the original repo less likely to be accepted since it changes the existing interface.